pkg/storaged/utils.js

```diff
@@ -199,18 +199,26 @@ export function lvol_name(lvol) {
     return cockpit.format('$0 "$1"', type, lvol.Name);
 }
 
-export function drive_name(drive) {
+function drive_main_name(drive) {
     const name_parts = [];
     if (drive.Vendor)
         name_parts.push(drive.Vendor);
     if (drive.Model)
         name_parts.push(drive.Model);
 
-    let name = name_parts.join(" ");
-    if (drive.Serial)
-        name += " (" + drive.Serial + ")";
-    else if (drive.WWN)
-        name += " (" + drive.WWN + ")";
+    return name_parts.join(" ");
+}
+
+function drive_sub_name(drive) {
+    return drive.Serial || drive.WWN;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)